draft-packages/empty-state/KaizenDraft/EmptyState/EmptyState.tsx

```diff
@@ -96,11 +106,12 @@ export const EmptyState: React.VFC<EmptyStateProps> = ({
       <div className={styles.textSide}>
         <div className={styles.textSideInner}>
           <Heading
-            variant="heading-3"
+            variant={headingProps.variant}
             classNameOverride={styles.heading}
-            tag="div"
+            tag={headingProps.tag}
+            color={headingProps.color}
           >
-            {headingText}
+            {headingProps.children || headingText}
```

Comment:
Just caught up with Cass and she made a good recommendation around passing the props directly into a Heading component like below:

```
          {headingProps ? (
            <Heading classNameOverride={styles.heading} {...headingProps} />
          ) : (
            <Heading
              variant="heading-3"
              classNameOverride={styles.heading}
              tag="div"
            >
              {headingText}
            </Heading>
          )}
```

This way we don't need to define default props on ln 64 and it makes it easier to untangle the deprecated `headingText` from the new `headingProps`.

Reply:
Kewl, thanks. Just pushed that change 👍